tests/xtro-sharpie/xtro-sanity/Sanitizer.cs

```diff
@@ -177,6 +177,10 @@ static void NoFixedTodo ()
 					foreach (var failure in failures)
 						sanitized.Remove (failure);
 					File.WriteAllLines (file, sanitized);
+					// since we are in AUTO_SANITIZE, if the file is empty, remove it.
+					if (!(File.ReadLines(file).Count() > 0)) {
```

Comment:
wrong style, space before `(`

Reply:
why not simply look if `sanitized` is empty ?
instead of reading back the file (that was just written)